FR/asciidoc/src/12_glossary.adoc

```diff
@@ -0,0 +1,42 @@
+ifndef::imagesdir[:imagesdir: ../images]
+
+[[section-glossary]]
+== Glossaire
+
+[role="arc42help"]
+****
+.Contenu
+Les termes techniques et métier les plus importants que vos parties prenantes utilisent lorsqu'elles discutent du système.
+
+Le glossaire peut également servir de source pour les traductions si vous travaillez dans des équipes multilingues.
+
+.Motivation
+Vous devez définir clairement vos termes, de manière à ce que toutes les parties prenantes
+
+* aient une compréhension identique de ces termes
+* n'utilisent pas de synonymes et d'homonymes
+
+
+.Représentation
+
+Un tableau avec les colonnes <Terme> et <Définition>.
+
+Potentiellement plus de colonnes au cas où vous auriez besoin de traductions.
+
+
```

Comment:
Maybe, you could add a table here. IMHO it makes it easier for the reader to understand how a glossary could look like, something like:

```
[options="header",cols="1,1"]
|===
|Terme|Définition
| _<Term-1>_ | _<Définition-1>_
| _<Term-2>_ | _<Définition-2>_ | _<Solution-2>_ | _<Lien-2>_
|===
```